Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -128,9 +128,25 @@ public interface IGitModule
         string GetEffectiveSetting(string setting);
         string GetEffectiveGitSetting(string setting, bool cache = true);
 
-        /// <summary>Gets the current branch; or "(no branch)" if HEAD is detached.</summary>
+        /// <summary>
+        /// Gets the name currently checked out branch.
+        /// </summary>
+        /// <returns>
+        /// The name of the branch (ex: main)
+        /// or "(no branch)" if HEAD is detached.
+        /// </returns>
         string GetSelectedBranch();
 
+        /// <summary>Gets the current branch; or .</summary>
+        /// <summary>
+        /// Gets the name currently checked out branch.
+        /// </summary>
+        /// <returns>
+        /// The name of the branch (ex: main)
+        /// or if HEAD is detached, string.Empty if setDefaultIfEmpty is true and "(no branch)" otherwise.
```

Comment:
```suggestion
        /// The name of the branch (for example: "main"); <see cref="string.Empty"/> or "(no branch)" depending on <param name="setDefaultIfEmpty"/>, if HEAD is detached.
```

Writing from memory, please check the syntax of `<param` tag in VS.

Reply:
Made me realized that I forgot the <param> tag so I have tweaked a little. A last look would be good...